recipes/mrio_common_metadata/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mrio_common_metadata" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mrio_common_metadata-{{ version }}.tar.gz
+  sha256: 0d315da7262d3f064346325e4e6739d7cdec255c263d2ce7d54ae3e9732e757c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - openpyxl
+    - pandas
+    - python >=3.6
+    - pyxlsb
+    - scipy
+    - xlrd
+
+test:
+  imports:
+    - mrio_common_metadata
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/brightway-lca/mrio_common_metadata
+  summary: Common Datapackage schema and utilities for MRIO tables
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - cmutel
```

Comment:
@cmutel, one more for you to confirm that you're willing to maintain.

Reply:
Confirmed!